clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -181,7 +181,15 @@ func (p *profileMeasurement) gatherProfile(c clientset.Interface) ([]measurement
 		return nil, fmt.Errorf("profile gathering failed finding component port: %v", err)
 	}
 	profileProtocol := getProtocolForComponent(p.config.componentName)
-	getCommand := fmt.Sprintf("curl -s -k %slocalhost:%v/debug/pprof/%s", profileProtocol, profilePort, p.config.kind)
+
+	var getCommand string
+	if p.config.componentName == "etcd" {
+		etcdCert := "/etc/srv/kubernetes/pki/etcd-apiserver-server.crt"
```

Comment:
Now we confirmed that it works we should create a flags for these two variables and default them to these values. The logic then should be: if flags are set, pass them and use https, if not don't pass and use http

Reply:
Use flags or env variables (`ETCD_CERTIFICATE` and `ETCD_KEY`) like https://github.com/kubernetes/perf-tests/blob/39b98208736db08b243b23ff53f1215c022cfb34/clusterloader2/pkg/measurement/common/etcd_metrics.go#L203 ?

Reply:
Let's use flags. We can use the "env-var" flag for convenience (can be set both via flag and env variable). The advantage is the basically a flag is clearly defined and documented (e.g. I can run --help and check what it does)

Reply:
> Now we confirmed that it works we should create a flags for these two variables and default them to these values. The logic then should be: if flags are set, pass them and use https, if not don't pass and use http

I create 2 flags but how can I get theses values into `clusterloader2/pkg/measurement/common/profile.go ` ?
https://github.com/kubernetes/perf-tests/pull/1136/commits/8e492ada85e2038ccebdb88b160ea306da0268da